src/extensions/uptime.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+
+import arc
+
+start_time = datetime.now()
+
+plugin = arc.GatewayPlugin("Blockbot Uptime")
+
+@plugin.include
+@arc.slash_command("uptime", "Show formatted uptime of Blockbot")
+async def uptime(ctx):
+    up_time = datetime.now() - start_time
+    h, ms = divmod(up_time.seconds, 3600)
+    m, s = divmod(ms, 60)
+    await ctx.respond(f"Uptime: **{up_time.days} days, {h} hours, {m} minutes, and {s} seconds**.")

```

Comment:
I did something like this, which handles plurals and omits 0 values:
```suggestion
    d = up_time.days
    h, ms = divmod(up_time.seconds, 3600)
    m, s = divmod(ms, 60)

    format = lambda val, str: f"{val} {str}{'s' if val != 1 else ''}"
    message_parts = [(d, "day"), (h, "hour"), (m, "minute"), (s, "second")]
    formatted_parts = [format(val, str) for val, str in message_parts if val]
    
    await ctx.respond(f"Uptime: **{', '.join(formatted_parts)}**")
```
![image](https://github.com/redbrick/blockbot/assets/61070210/abffe39d-190c-480d-b5fe-3dcf0d89ef27)

Reply:
it's a bit longer _but_  the timestamp's more naturally worded - i like it 

<img src="https://img-03.stickers.cloud/packs/2150b09e-9151-46fc-8e43-1b3f40aebb50/webp/f248964e-eb62-4f37-86b2-277fc7ef36c4.webp" height="100px">

Reply:
Longer isn't an issue in most cases, just readability by design! :+1: